survival_guide/1_concepts.adoc

```diff
@@ -23,7 +23,13 @@ This topic covers some of the most important concepts of the Fusion cluster depi
 
 //tag::which[]
 
-Although Figure 1 above depicts running in Google Kubernetes Engine (GKE), Fusion 5 runs on any modern version of Kuberentes (v 1.14 or newer). In fact, very little code in Fusion has any awareness of Kubernetes or Docker. The K8s ecosystem is moving very fast and infrastructure providers are quickly offering some flavor of Kubernetes. However, the beauty of Kubernetes is that applications like Fusion do not need to care about the underlying infrastructure. Although Lucidworks provides setup scripts for GKE, AKS, and EKS to help users get started with Fusion 5, do not take this to mean Fusion only runs on those flavors of Kubernetes. In general, as long as K8s is running, Fusion 5 can run on it.
+Figure 1 above depicts running in Google Kubernetes Engine (GKE). To run Fusion 5, use your cloud provider's stable release channel.
+
+NOTE: GKE refers to the version as *stable release channel*. Other providers may use a different term for the version on which to run Fusion 5. 
```

Comment:
I feel like we should be more helpful here.  I'd suggest making the "release channel" text a hyperlink to https://cloud.google.com/kubernetes-engine/docs/concepts/release-channels

We can also include references for the other providers' support calendars, which should update as their versions change (we wouldn't have to change these links).
EKS: https://docs.aws.amazon.com/eks/latest/userguide/kubernetes-versions.html#kubernetes-release-calendar
AKS: https://docs.microsoft.com/en-us/azure/aks/supported-kubernetes-versions#aks-kubernetes-release-calendar

Reply:
Added the links and will push the changes to the PR.

Reply:
How about this for the second sentence?

> Your cloud provider may use different terminology to indicate the latest stable version of Kubernetes that is available on their platform.

Reply:
Second sentence replaced.